SKYKitChat/Classes/UI/ParticipantList/SKYChatParticipantListViewController.swift

```diff
@@ -0,0 +1,307 @@
+//
+//  SKYChatParticipantListViewController.swift
+//  SKYKitChat
+//
+//  Copyright 2016 Oursky Ltd.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import SVProgressHUD
+
+public enum SKYChatParticipantQueryMethod: UInt {
+    // Initial value for the enum, for compatible with objective-c
+    case Undefined
+
+    // By username of the user, supposed to be unique
+    case ByUsername
+
+    // By email of the user, supposed to be unique
+    case ByEmail
+
+    // By name of the user, can limit the searching scope by setting participantScope
+    case ByName
+}
+
+open class SKYChatParticipantListViewController: UIViewController {
+
+    static let queryMethodCoderKey = "QUERY_METHOD"
+
+    public var skygear: SKYContainer?
```

Comment:
Together with [this comment](https://github.com/SkygearIO/chat-SDK-iOS/pull/11/files/9df47b21a9ccc7cab65e46fbde912fe919464527#r96564272), they come with a decision to be made: whether we require the user to set something, i.e. making a subclass of the classes in UIKit, or not. If we provide a default value for both `self.skygear` and `self.queryMethod`, users would no need to making a subclass.

Any ideas ?